hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystemMetrics.java

```diff
@@ -81,9 +84,10 @@ public static void init() throws Exception {
         .setStreamBufferMaxSize(4) // MB
         .build();
     cluster.waitForClusterToBeReady();
+    client = cluster.getClient();
 
     // create a volume and a bucket to be used by OzoneFileSystem
-    bucket = TestDataUtil.createVolumeAndBucket(cluster);
+    bucket = TestDataUtil.createVolumeAndBucket(client);
```

Comment:
Programmatically, the `TestDataUtil.createVolumeAndBucket` creates and `closes` `OzoneClient` internally is a better model. It'll be much less noisy.

Reply:
Previously, `cluster` was passed and the problem is `createVolumeAndBucket` didn't close the client it acquires internally. Just fixing that would result in less code change than allowing `client` to be passed (and many test classes have to introduce the client variable and ensure closing it in this PR). 
The test class can create a new `client` if for their needs and ensure closing themself.

But, I'm ok either way.

Reply:
I just remembered another reason for doing it this way: the method returns `OzoneBucket`, which some tests ignore, but some tests keep using it.  `OzoneBucket` internally stores the client.  If we close the client in `createVolumeAndBucket`, the object will be useless.